R/get_url.R

```diff
@@ -0,0 +1,48 @@
+# get_url_bioconductor_package -------------------------------------------------
+get_url_bioconductor_package <- function(package) {
+
+  get_url_bioconductor_package_2("https://bioconductor.org", package)
+}
+
+# get_url_bioconductor_package_2 -----------------------------------------------
+get_url_bioconductor_package_2 <- function(base_url, package) {
```

Comment:
could these two BioConductor URL functions be merged into one or get more informative names?

Reply:
Yes. However, I would like to do this later. I would like to work over the other code files from which I would like to extract all URL generating parts in a similar manner. Once this is done, and once it is clear in which ways URLs are put together, we can look over the helper functions again and combine/rename them as appropriate.

Reply:
Fine, it makes sense!